package.json

```diff
@@ -41,7 +41,7 @@
     "isstream": "~0.1.1"
   },
   "scripts": {
-    "test": "npm run lint && node node_modules/.bin/taper tests/test-*.js && npm run test-browser && npm run clean",
+    "test": "npm run lint && node node_modules/.bin/taper tests/test-*.js && npm run test-browser",
```

Comment:
I think we had a clean script back then, that logic was later moved to the browser test itself. NPM on node 0.10 doesn't complain about the missing script, where the newer version of it throws an error.